packages/react-devtools-shared/src/devtools/views/Profiler/Schedulers.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import type {CommitTree} from './types';
+import type {SerializedElement} from '../Components/types';
+
+import * as React from 'react';
+import {useContext} from 'react';
+import {ProfilerContext} from './ProfilerContext';
+import styles from './Schedulers.css';
+
+export type Props = {|
+  commitTree: CommitTree,
+  updaters: Array<SerializedElement>,
+|};
+
+export default function Schedulers({commitTree, updaters}: Props) {
```

Comment:
A different term for the same UI or is there a difference between "schedulers" and "updaters"?

nit:
```suggestion
export default function Updaters({commitTree, updaters}: Props) {
```

Reply:
Ah, well it's the thing that "_scheduled_ the _update_" 😁 but I could have been more consistent.